src/core/core.controller.js

```diff
@@ -265,7 +268,7 @@ class Chart {
 		this._updating = false;
 		this.scales = {};
 		this.scale = undefined;
-		this.$plugins = undefined;
+		this._plugins = new PluginService();
```

Comment:
It's a bit unrelated, but should we rename `$proxies` to `_proxies` while we're at it?

Reply:
`$` is commonly (and in the `$proxies` case) used to indicate variable set from outside current scope, whereas `_` indicates a private (internal) variable.

Reply:
Leaving the `$` is fine with me then